plotnine/geoms/annotate.py

```diff
@@ -11,8 +12,8 @@ class annotate:
 
     Parameters
     ----------
-    geom : geom
-        Name of geom to use for annotation
+    geom_cls_or_name : geom
```

Comment:
No need to change the name of the parameter. Relevant type info in the comment is enough.

```
geom : geom or str
```